BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/ExceptionConverterTest.cs

```diff
@@ -185,6 +185,19 @@ public void DoesNotTrimShortExceptionMessages()
             Assert.AreEqual(5, expDetails.message.Length);
         }
 
+        [TestMethod]
+        [DataRow(null)]
+        [DataRow("")]
+        [DataRow(" ")]
+        public void ProvidesDefaultMessage(string message)
```

Comment:
`message` parameter is not used within this method.

Reply:
My bad, started with just an empty string test then decided to add `DataRow`s to cover null and whitespace too. I've fixed the test to use the parameter now.